source/common/src/tmsg.c

```diff
@@ -6908,6 +6910,23 @@ int32_t tDecodeSVAlterTbReq(SDecoder *pDecoder, SVAlterTbReq *pReq) {
     default:
       break;
   }
+  return 0;
+}
+
+int32_t tDecodeSVAlterTbReq(SDecoder *pDecoder, SVAlterTbReq *pReq) {
+  if (tStartDecode(pDecoder) < 0) return -1;
+  if (tDecodeSVAlterTbReqCommon(pDecoder, pReq) < 0) return -1;
+  if (tDecodeI64(pDecoder, &pReq->ctimeMs) < 0) return -1;
```

Comment:
If you want to ensure compactibility, write code like below:

if (tDecodeIsEnd(pDecoder)) {
  if (tDecodeI64(pDecoder, &pReq->ctimeMs) < 0) return -1;
} else {
  pReq->ctimeMs = 0; /* some default value */
}

Reply:
fixed